lib/node_modules/@stdlib/stats/base/dists/betaprime/mean/README.md

```diff
@@ -144,6 +144,98 @@ for ( i = 0; i < 10; i++ ) {
 
 <!-- /.examples -->
 
+<!-- C interface documentation. -->
+
+* * *
+
+<section class="c">
+
+## C APIs
+
+<!-- Section to include introductory text. Make sure to keep an empty line after the intro `section` element and another before the `/section` close. -->
+
+<section class="intro">
+
+</section>
+
+<!-- /.intro -->
+
+<!-- C usage documentation. -->
+
+<section class="usage">
+
+### Usage
+
+```c
+#include "stdlib/stats/base/dists/betaprime/mean.h"
+```
+
+#### stdlib_base_dists_betaprime_mean( alpha, beta )
+
+Evaluates the mean for a beta prime distribution with first shape parameter `alpha` and second shape parameter `beta`.
+
+```c
+double y = stdlib_base_dists_betaprime_mean( 1.0, 2.0 );
+// returns ~1.0
+```
+
+The function accepts the following arguments:
+
+-   **alpha**: `[in] double` first shape parameter
+-   **beta**: `[in] double` second shape parameter
+
+```c
+double stdlib_base_dists_betaprime_mean( const double alpha, const double beta );
+```
+
+</section>
+
+<!-- /.usage -->
+
+<!-- C API usage notes. Make sure to keep an empty line after the `section` 
+element and another before the `/section` close. -->
+
+<section class="notes">
+
+</section>
+
+<!-- /.notes -->
+
+<!-- C API usage examples. -->
+
+<section class="examples">
+
+### Examples
+
+```c
+#include "stdlib/stats/base/dists/betaprime/mean.h"
+#include "stdlib/constants/float64/eps.h"
+#include <stdlib.h>
+#include <stdio.h>
+
+int main( void ) {
+    double alpha;
+    double beta;
+    double y;
+    int i;
+
+    for ( i = 0; i < 10; i++ ) {
+        alpha = ( ( (double)rand() / (double)RAND_MAX )*10.0 ) + STDLIB_CONSTANT_FLOAT64_EPS;
```

Comment:
We should define 

```c
static double random_uniform( const double min, const double max ) {
    double v = (double)rand() / ( (double)RAND_MAX + 1.0 );
    return min + ( v*(max-min) );
}
```

and use it to generate the parameters for added clarity